modules/core/src/main/java/org/apache/ignite/events/EventType.java

```diff
@@ -922,6 +922,16 @@
      */
     public static final int EVT_CLUSTER_SNAPSHOT_FAILED = 151;
 
+    /**
+     * Built-in event type: query execution.
```

Comment:
Could you please provide a comprehensive description of this event?
The current Javadoc gives no clue on this event. Moreover, `SqlQueryExecutionEvent` itself does not provide any specific information too.
At least, it would be useful to clearly state the phase of query execution that triggers this event and etc.

Reply:
Thanks, @sk0x50!
Your patch for this issue is welcome.

> SqlQueryExecutionEvent

What kind of specific information do you have in mind?
For now, we have text, type, user, args.
It's enough for any king of event handling I can imagine.

Reply:
Hello @nizhikov ,

> What kind of specific information do you have in mind?
At least, it would be useful to clearly state the phase of query execution that triggers this event.
Currently, this important information cannot be obtained via Javadoc. So this the reason for kindly asking @SomeFire (as a contributor) and @nizhikov (as a reviewer) to provide something useful instead of `Query execution event.` :)

Reply:
Perhaps, something as follows:
```
This event is triggered after a corresponding SQL query validated and before it is executed.
Unlike {@link #EVT_CACHE_QUERY_EXECUTED}, EVT_SQL_QUERY_EXECUTION is fired only once for a request and does not relate to a specific cache.
```

What do you think? Is it good enough?

Reply:
Sounds good